monkestation/code/modules/cargo/packs.dm

```diff
@@ -1,3 +1,103 @@
+//////////////////////////////////////////////////////////////////////////////
+//////////////////////////// Armory //////////////////////////////////////////
+//////////////////////////////////////////////////////////////////////////////
+
+/datum/supply_pack/security/armory/revolver
+	name = "Revolver Single-Pack"
+	desc = "Contains one revolver, of the sort detectives are known to habitually smuggle on board.  Ammo not included."
```

Comment:
I think the revolver actually comes pre-loaded

Reply:
fixed